yahoofantasy/resources/league.py

```diff
@@ -28,6 +28,21 @@ def teams(self, persist_ttl=DEFAULT_TTL):
             teams.append(t)
         return teams
 
+    def players(self, persist_ttl=DEFAULT_TTL):
+        logger.debug("Looking up players")
+        START = 0
+        COUNT = 25
+        data = self.ctx._load_or_fetch(f"players.{self.id}.{START}", f"players;count={COUNT};start={START}", league=self.id)
```

Comment:
I created a custom persistance path to handle players collection.

These load or fetch lines has more than 79 characters, I can format them to comply with PEP8.